XMonad/Doc/Extending.hs

```diff
@@ -1156,6 +1156,9 @@ A non complete list with a brief description:
     A module for abstracting a font facility over
     Core fonts and Xft.
 
+* "XMonad.Util.Hacks":
```

Comment:
I feel like this could be made more prominent and easier to discover than the other X.Util modules. An additional mention of it somewhere or something. (Maybe it's premature now that it only contains one little hack.)

Reply:
I think it'd definitely be to early to make that "big" right now, given that it really _is_ completely empty except for that one thing. We should wait until a couple more things are in there until we start making it more prominent, imo.